src/app/integrations/list/list.component.html

```diff
@@ -108,8 +108,8 @@ <h4 class="modal-title pull-left">Confirm {{ getActionTitle() }}</h4>
 
           <div *ngIf="integration.currentStatus !== 'Pending'"
                class="not-pending">
-              <span class="label label-{{ getLabelClass(integration.currentStatus) }}"
-                    disabled>{{ getStatusText(integration.currentStatus) }}</span>
+            <span class="label label-{{ getLabelClass(integration.currentStatus) }}"
```

Comment:
Gah! Those formatting changes. Not your fault, we just need to get on the same page. I don't mind either way but maybe we could throw it in the community site.